decidim-core/db/migrate/20210210114657_add_external_domain_whitelist_to_organization.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class AddExternalDomainWhitelistToOrganization < ActiveRecord::Migration[5.2]
+  def change
+    add_column :decidim_organizations, :external_domain_whitelist, :string, array: true, default: []
+
+    reversible do |direction|
+      direction.up do
+        Decidim::Organization.find_each do |organization|
```

Comment:
We could also use `update_all`:
https://api.rubyonrails.org/v6.1.1/classes/ActiveRecord/Relation.html#method-i-update_all

Reply:
Linter says "avoid using `update_all` because it skips validations."

Reply:
OK, let's keep it as is. Ignore.

Reply:
Sorry to reopen this, but I think it would be better to use `update_all` here, for several reasons:
* It is widely used on Decidim's migrations. You should use a comment to disable the linter on that line. Here you have some examples to check: https://github.com/decidim/decidim/search?q=SkipsModelValidations
* It prevents migrations to fail if there is any validation error during the update process.
* It avoids the load of model instances during migrations. This also could raise errors during the migrations because the code model used will be the current one, but the database version will be outdated, with some missing some columns or tables. If you really need to use a model (not in this case), it's better to redefine the class locally, and not use the real one. You can see some nice examples of this [here](https://github.com/decidim/decidim/blob/e015b8d2374463c57c88d9cd254872d94f661f16/decidim-core/db/migrate/20170605162500_add_hierarchy_to_scopes.rb) or [here](https://github.com/decidim/decidim/blob/fd99b8a13c503cafe421289005db373ed4b187bf/decidim-initiatives/db/migrate/20171017095143_update_initiative_scoped_type.rb).